### ISSUE TITLE
eslint changed action note

### ISSUE THREAD

**Issue by bradennapier** [frontity/frontity#517]
I am not sure it will work for you at the moment as you are using `if: ${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository }}` which would tell it to not run at all if its from a forked repo, no?

Also, just updated to `v3.1.0` which fixes some issues that you may never encounter but.... :-)

**Comment by luisherranz** [comment]
> I am not sure it will work for you at the moment as you are using if: \${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository }} which would tell it to not run at all if its from a forked repo, no?

You're right! Thanks for the catch @bradennapier 🙂

I've fixed it in https://github.com/frontity/frontity/commit/ad52f9f7496e7c70e4d20fc01b04bb66af9bd798.

> Also, just updated to v3.1.0 which fixes some issues that you may never encounter but.... :-)

Shouldn't you move the `v3` tag to point to the last stable version?

**Comment by bradennapier** [comment]
If I do that kind of thing the marketplace listing gets messed up, but I will try.  They don't really support multiple release tags well atm. It is `3.4.0` now.  Let me try again though.

We will likely prefer the semantic release moving forward though - github is also moving this way.

**Comment by luisherranz** [comment]
I see. And what happens when there are hundreds of projects using a tag of a GH action, for example `v1`, and a critical bug appears. Do all those projects need to update to `v1.0.1` in their workflows?

**Comment by bradennapier** [comment]
Not sure how they will do it - I know that the `setup-node` broke for everyone using `v2` and had to change to `2.0.0` lol.  They clearly have a decent number of improvements to make to github actions.... but i do like them still :-)

**Comment by luisherranz** [comment]
Yes, it looks like they recommend moving the "major" tag to the new latest commit: https://docs.github.com/en/actions/creating-actions/about-actions#using-release-management-for-actions. If you faced problems doing so I guess it'd be because of a bug 🤷
